plugins/swift/openapi_swift_generator/Sources/openapi_swift_generator/main.swift

```diff
@@ -0,0 +1,42 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+func Log(_ message : String) {
+  FileHandle.standardError.write((message + "\n").data(using:.utf8)!)
+}
+
+func main() throws {
+  
+  // read the OpenAPI document
+  let rawRequest = try Stdin.readall()
+  let request = try Openapi_Plugin_V1_Request(protobuf: rawRequest)
+  let wrapper = request.wrapper
+  let document = try Openapi_V2_Document(protobuf:wrapper.value)
+
+  // build the service renderer
+  let renderer = ServiceRenderer(document:document)
+
+  // generate the desired files
+  var response = Openapi_Plugin_V1_Response()
+  let filenames = ["client.swift", "service.swift", "types.swift"]
```

Comment:
Any reason why this is different than the other sample where client and server were conditionally generated ?
https://github.com/googleapis/openapi-compiler/blob/master/plugins/go/openapi_go_generator/main.go#L51